airbyte-integrations/connectors/source-looker/sample_files/full_configured_catalog.json

```diff
@@ -4765,6 +4765,27 @@
       },
       "sync_mode": "full_refresh",
       "destination_sync_mode": "overwrite"
-    }
+    },
+    {
+      "stream": {
+        "name": "run_looks",
+        "json_schema": {
+          "$schema": "http://json-schema.org/draft-07/schema#",
+          "additionalProperties": true,
+          "type": "object",
+          "properties": {
+            "look_id": {
+              "properties": {
+                "additionalProperties": true
+              },
+              "type": ["null", "object"],
+              "additionalProperties": true
+            }
+          }
+        },
+        "supported_sync_modes": ["full_refresh"],
+        "destination_sync_mode": "overwrite",
+        "source_defined_cursor": false
+      }
   ]
```

Comment:
hmm, isn't this square bracket required?

Reply:
This bracket is marked as red because a `}` is missing before it.